components/backends/trtllm/launch/agg.sh

```diff
@@ -19,12 +19,12 @@ trap cleanup EXIT INT TERM
 # run clear_namespace
 python3 utils/clear_namespace.py --namespace dynamo
 
-# run ingress
-dynamo run in=http out=dyn --http-port=8000 &
+# run frontend
+python3 -m dynamo.frontend --http-port 8000 &
 DYNAMO_PID=$!
 
 # run worker
-python3 components/worker.py \
+python3 -m dynamo.trtllm \
```

Comment:
not for this PR - but question that will probably come up later - do you think these should be `dynamo.backends.{trtllm,sglang,vllm,llama_cpp}` etc. instead of `dynamo.{backend}`?

Reply:
Yes that's how we originally designed it BUT it means `components/backends/trtllm/src/dynamo/backend/trtllm/main.py`. It's pretty awful path-wise. It's also a sub-module split into multiple directories. We're not going to have double digits backends, so I figure it's less typing.

Reply:
Can we construct the layered directory when we build the package? So that at source code level it is still `src/dynamo/trtllm` but we can call `dynamo.backend.trtllm` when accessing the module